rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -6,4 +6,12 @@ recipeList:
   - org.openrewrite.java.ChangePackage:
       oldPackageName: com.azure
       newPackageName: io.clientcore
-      recursive: true
\ No newline at end of file
+      recursive: true
+  - org.openrewrite.text.FindAndReplace:
+      find: 'requires com.azure.core'
+      replace: 'requires com.azure.core.v2'
+      filePattern: '**/module-info.java'
+  - org.openrewrite.text.FindAndReplace:
+      find: 'requires transitive com.azure.core'
```

Comment:
Is it possible to use regex here to do something like `requires (transitive )?com.azure.core` instead of having two recipes?